index.html

```diff
@@ -19,22 +19,30 @@
         <link
             href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
             rel="stylesheet">
+            <script src="script.js"></script>
 </head>
 
 <body>
 <main>
     <!----------navbar setion------------------->
     <header>
     <nav class="nav-container">
-        <a class="Johnd" href=""></a>
-        <a class="menu" href="#"></a>
-        <ul>
-            <li><a href="">Portfolio</a></li>
-            <li><a href="">About</a></li>
-            <li><a href="">Contact</a></li>
-            <li><a href=""><img src="/images/msg.png"></a></li>
+       
+        <a class="Johnd" href="#"></a>
+        <a class="menu" href="#" onclick="displayPopUp()"></a>
+        <ul class="menu-items">
+            <li><a href="#my-works">Portfolio</a></li>
+            <li><a href="#about-me">About</a></li>
+            <li><a href="#contact-me">Contact</a></li>
+            <li><a href="#"><img src="./images/msg.png"></a></li>
         </ul>
     </nav>
+    <ul class="popup-menu" id="popup-menu-id">
+        <li class="cross" id="cross-id"><a href=""><img src="./images/cross.png" onclick="closePopUp()"></a></li>
+        <li id="portFolio-id" ><a href="#my-works" onclick="closePopUp()">Portfolio</a></li>
+        <li id="aboutMe-id" ><a href="#about-me" onclick="closePopUp()">About</a></li>
+        <li id="contactMe-id"><a href="#contact-me" onclick="closePopUp()">Contact</a></li>
+    </ul>
```

Comment:
## OPTIONAL


- [ ] It would be better for the background not to be scrollable when the mobile nav is open, this will create a better user experience  👍🏿

Reply:
yes, surely will add this css property for better user exoerience